src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleModel.scala

```diff
@@ -1,16 +1,24 @@
 package org.broadinstitute.dsde.workbench.sam.google
 
 import cats.effect.IO
-import org.broadinstitute.dsde.workbench.model.{ErrorReport, ErrorReportSource}
+import org.broadinstitute.dsde.workbench.model.{ErrorReport, ErrorReportSource, WorkbenchEmail, WorkbenchIdentityJsonSupport}
 import spray.json.DefaultJsonProtocol
 
 object SamGoogleModelJsonSupport {
   import DefaultJsonProtocol._
   import org.broadinstitute.dsde.workbench.model.ErrorReportJsonSupport._
+  import WorkbenchIdentityJsonSupport._
 
   implicit val SyncReportItemFormat = jsonFormat4(SyncReportItem.apply)
+  implicit val SyncedPolicyFormat = jsonFormat2(SyncedPolicy.apply)
+  implicit val SyncReportFormat = jsonFormat1(SyncReport.apply)
+
 }
 
+case class SyncReport(syncedPolicies: Seq[SyncedPolicy])
```

Comment:
Is there a reason we need this out layer? I don't have a strong opinion either way, it just doesn't seem to add anything.

Reply:
No real reason, except to make the top layer be an object rather than array. Opens up the possibility of a sync summary, where successes/failures could be counted, or any other metadata.